recipes/cafe/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "4.2.1" %}
+{% set sha256 = "f58da8f79eda33c233f0bddaa0f780eaa7fdb3b874d8841f0a7f2cba3cc3533f" %}
+
+package:
+  name: cafe
+  version: {{ version }}
+
+source:
+  url: https://github.com/hahnlab/CAFE/releases/download/v{{ version }}/CAFE-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+test:
+  commands:
+    - which cafe
+
+about:
+  home: https://hahnlab.github.io/CAFE/
+  license: IU OPEN SOURCE LICENSE (see https://github.com/hahnlab/CAFE/blob/master/LICENSE)
+  license_file: LICENSE
```

Comment:
This reference the file in the tarball ... https://github.com/hahnlab/CAFE/blob/v4.2.1/LICENSE

So I assume you can remove the copy of license file in this PR.

Reply:
Thanks for the advice, I've remove the copy of license file in this PR.